src/tasks/deploy.mk

```diff
@@ -1,6 +1,8 @@
 #Must be above deplo%
 deploy-asset%:
-	@if [ -e public/asset-hashes.json ]; then nht deploy-hashed-assets --monitor-assets; fi
+	@if [ -e public/asset-hashes.json ] && [ -e anvil.config.js ]; then nht deploy-hashed-assets --monitor-assets --app anvil; fi
+	@if [ -e public/asset-hashes.json ] && [ -e n-ui-build.config.js ]; then nht deploy-hashed-assets --monitor-assets; fi
```

Comment:
I'm not that familiar with `n-ui`, but is `n-ui-build.config.js` currently being used or is it going to be introduced with anvil?

Reply:
it's used by n-ui but i'm pretty sure it's optional https://github.com/search?q=org%3Afinancial-times+filename%3An-ui-build.config.js

Reply:
If it's optional, does it need the `[ -e n-ui-build.config.js ];` part? Can it just be an `else if` (or equivalent) for line 3?

Reply:
Yep this was avoiding `else if` because I don't know how to do it 😆 

I've used `if` blocks before over multiple lines but I can't do this, can I?

```make
if [ -e public/asset-hashes.json ]
  if [ -e anvil.config.js ]
    nht deploy-hashed-assets --monitor-assets --app anvil
  else
    nht deploy-hashed-assets --monitor-assets
  endif
endif
```

Reply:
you can if you backslash immediately before the newline

Reply:
That appears to require mixing spaces and tabs... bit scary? Or this, maybe:

```makefile
ifeq (, $([-e public/asset-hashes.json]))
ifeq (, $([-e anvil.config.js]))
	nht deploy-hashed-assets --monitor-assets --app anvil
else
	nht deploy-hashed-assets --monitor-assets
endif
endif
```

Reply:
Mixed tabs and spaces one...

```makefile
	@if [ -e public/asset-hashes.json ]; then\
    if [ -e anvil.config.js ]; then\
      nht deploy-hashed-assets --monitor-assets --app anvil;\
    else\
      nht deploy-hashed-assets --monitor-assets;\
    fi;\
  fi
```

Reply:
Mixed tabs and spaces one...

```makefile
	@if [ -e package.json ]; then\
    if [ -e readme.md ]; then\
      echo 'ok';\
    else\
      echo 'cool';\
    fi;\
  fi
```

Reply:
I like the `ifeq` version more. Seems more readable.

What does the `,` do? ie, `(, $([-e public/asset-hashes.json]))`

Reply:
I don't know exactly, but my understanding from trying to parse https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html is that it always requires two statements. 

Either way, it doesn't actually work! 🤔

Reply:
Another way I've done `if` statements in Make is with

```
$(if [condition],\
  [if-command], \
  [else-command] \
)
```

eg, https://github.com/Financial-Times/n-gage/blob/56b4f7d45bb5115eeb248f56837ce5d84a52d1c3/src/tasks/review-app.mk#L27-L30

You'd need to figure out the `[condition]` part for checking the file though.